asciidoc/courses/app-typescript/modules/1-driver/lessons/5-c-writing-data/lesson.adoc

```diff
@@ -49,19 +49,83 @@ Run the file using `ts-node` to view the result:
 ts-node {lab-file}
 ----
 
-You can check the result in the Neo4j Browser by running the following query:
+If successful, you should see output similar to the following (with your name instead of "Your Name"):
+
+.Expected Output
+[source,json,role=nocopy]
+----
+{
+  identity: Integer { low: 12345, high: 0 },
+  labels: [ 'Person' ],
+  properties: { name: 'Your Name' },
+  elementId: '4:abc123:12345'
+}
+----
+
+You can also verify the result in the Neo4j Browser by running the following query:
 
 [source, cypher]
 ----
 MATCH (m:Movie {title: "Matrix, The"})<-[a:ACTED_IN]-(p:Person)
 RETURN m, a, p
 ----
 
+You should see your `Person` node connected to _The Matrix_ movie.
+
+
 include::questions/verify.adoc[leveloffset=+1]
 
+
+== Troubleshooting
+
+.ts-node: command not found
```

Comment:
Troubleshooting - good idea for complex activities. Used as a last resort, we should design the course content so that these scenarios are few, and prepare the learner to be able to solve their own issues.

Im not sure about giving multiple options for fixing an issue:

. use ts-node
. or npx

It prompts questions for the learner:
- Which one should I use? 
- Is one better than the other?
- Why am I experiencing this problem?

I would direct them back to the installation instructions in the course which should be comprehensive / robust enough to cover the majority of scenarios.